### PR TITLE
WIP Apply website redesign rfc

### PR DESCRIPTION
**Pull request by jenweber**
The goal of this PR is to minimally apply the website redesign styles found in ember-styleguide. See https://github.com/ember-learn/ember-styleguide/pull/145

It aims to remove more CSS than it adds, and serve as a blank canvas to apply more advanced styles to after initial launch.

This goes with this other PR https://github.com/ember-learn/guidemaker-ember-template/pull/30

## Added
- overrides for H4. They did not appear bold at all and they need to stand out.
- overrides to the `container` class so that content doesn't run up against the edges in mobile view

## Changed
- colors to match the styleguide

## Removed
- fontello
- a lot of CSS that could be replaced by utility classes

## Needs work to reach MVP
- default link color is not accessible for regular sized text
- The search dropdowns need a lot of work, probably.
- links in code blocks look weird
- hr looks weird
- why are footer assets missing in apps that use guidemaker?

## Temporary changes

I pinned to a specific SHA of ember-styleguide. This would need too be replaced with the correct published version.

## Screenshots

Screen Shot 2019-11-02 at 6 00 11 PM

Screen Shot 2019-11-02 at 6 06 20 PM

### CODE REVIEW

**Comment by wifelette** [comment]
Fantastic work Jen, thanks so much for getting the ball rolling on this bit, I'm excited to see it out in the wold!

**Comment by MelSumner** [comment]
@mansona are the Travis failures re: node versions still okay?

File: package.json [hunk 1]
```diff
@@ -24,28 +24,28 @@
...
+    "ember-cli-htmlbars": "^3.0.1",
     "ember-cli-sass": "^10.0.0",
     "ember-cli-showdown": "^4.4.4",
     "ember-collapsible-panel": "^3.1.1",
-    "ember-composable-helpers": "^2.1.0",
     "ember-href-to": "^3.1.0",
     "ember-power-select": "^2.0.9",
     "ember-prism": "^0.4.0",
-    "ember-styleguide": "^2.5.0",
+    "ember-styleguide": "4.0.0-10",
```
Comment (MelSumner) [inline package.json]:
```suggestion
    "ember-styleguide": "^4.0.0-14",
```

File: package.json [hunk 2]
```diff
@@ -24,28 +24,28 @@
...
+    "ember-cli-htmlbars": "^3.0.1",
     "ember-cli-sass": "^10.0.0",
     "ember-cli-showdown": "^4.4.4",
     "ember-collapsible-panel": "^3.1.1",
-    "ember-composable-helpers": "^2.1.0",
     "ember-href-to": "^3.1.0",
     "ember-power-select": "^2.0.9",
     "ember-prism": "^0.4.0",
-    "ember-styleguide": "^2.5.0",
+    "ember-styleguide": "^4.0.0-14",
```
Comment (MelSumner) [inline package.json]:
```suggestion
    "ember-styleguide": "4.0.0-10",
```

**Comment by mansona** [comment]
@MelSumner sorry things have been a bit busy for me this weekend so I didn't get a chance to share plans with everyone but I was discussing with @jessica-jordan and we should start doing PRs into this branch (like we did with the website redesign). So let's not commit directly to this branch unless you're doing a pre-release (like we are doing with styleguide) 👍

**Comment by jenweber** [comment]
Closed, we are opening a fresh branch shortly for this work!

**Pull request closed without merging**